internal/mill/exec.go

```diff
@@ -168,6 +169,52 @@ func (p *Petsitter) execPetsFileAt(t *skylark.Thread, module string, isMissingOk
 	return skylark.StringDict(result), nil
 }
 
+// Service(server, “localhost”, 8081)
+func (p *Petsitter) service(t *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	// Are we starting a process (service?) or modifying one?
+	var server skylark.Dict
+	var host string
+	var port int
+	var pr proc.PetsProc
+	var pkey skylark.Int
+	var pid int
+
+	if err := skylark.UnpackArgs("service", args, kwargs, "server", &server, "host", &host, "port", &port); err != nil {
+		return nil, err
+	}
+
+	// get pid from server as go object - get process of pid from procfs list (all in go)
+	for _, serverItem := range server.Items() {
```

Comment:
instead of iterating through the items, i think you can do server.Get(skylark.String("pid")), because that's the key you used in 'start'?